training/distributed_training/pytorch/model_parallel/gpt2/train_gpt_simple.py

```diff
@@ -97,7 +97,7 @@ def train_step(model, optimizer, input_ids, attention_mask, args):
     else:
         loss = model(input_ids=input_ids, attention_mask=attention_mask, labels=input_ids)["loss"]
     if args.fp16:
-        optimizer.backward(loss, update_master_grads=False)
+            optimizer.backward(loss, update_master_grads=False)
```

Comment:
There's more cases that I haven't pointed to as well, please review the file's indentation changes